app/views/professional_experiences/index.html.erb

```diff
@@ -0,0 +1,48 @@
+<div id="page-top" class="row">
+  <div class="container-fluid home">
+    <div class="card mb-3">
+      <div class="card-header">
+        <i class="fa fa-table"></i>
+        <%= t('activerecord.models.professinal_experience.other') %>
+      </div>
+      <div class="card-body">
+        <div class="table-responsive">
+          <table class="table table-hover table-bordered users-evaluation" id="dataTable">
+            <thead>
+              <tr>
+                <th><%= t('activerecord.attributes.professional_experience.company') %></th>
```

Comment:
you can use here human_attribute_name

https://api.rubyonrails.org/classes/ActiveModel/Translation.html#method-i-human_attribute_name

https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models